Jenkinsfile

```
@@ -85,7 +85,7 @@ pipeline {
                     steps {
                         checkPatch user: GITHUB_USER_USR,
                                    password: GITHUB_USER_PSW,
-                                   ignored_files: "src/control/vendor/*"
+                                   ignored_files: "src/control/vendor/*,*.pb.[ch]"
```

It seems like `,` would be a good separator and maybe should be accepted, but currently `:` is the only accepted separator.

```suggestion
                                   ignored_files: "src/control/vendor/*:*.pb.[ch]"
```